classes/DirtSoundLibrary.sc

```diff
@@ -320,8 +348,9 @@ DirtSoundLibrary {
 	}
 
 	makeEventForBuffer { |buffer, metaData|
-		var baseFreq = 60.midicps;
+		var baseFreq = StrudelUtils.baseNote().midicps;
```

Comment:
I intentionally kept separate `DirtSoundLibrary` and `SuperDirt`. This `StrudelUtils.baseNote().` couples both to one system. MAybe we find a way around this?